tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -106,17 +106,25 @@ class Foo
         // there should be no docblock here
         $this->title = $title;
     }
-}
 
-/*********************
- * Some long comment *
- *********************/
-// This is a one-line c++ style comment
-/* This is a multi line comment
-   yet another line of comment */
-/* This is a multiple-lines comment block
-that spans over multiple
-lines */
-// This is a one-line shell-style comment
+    /**
```

Comment:
I think we should't change comment output, print as is

Reply:
It is very bad practice

Reply:
you can tests how prettier handle comment using `playground`

Reply:
👍 I agree. The printComments function needs some more work, was just trying to get something to cover the most common cases for now but agreed its currently too opinionated